src/ocamlorg_frontend/pages/exercises.eml

```diff
@@ -1,23 +1,59 @@
+let exercises_sidebar_link
+~title
+~href
+~current =
+  match current with
+  | true ->
+    <a class="block text-primary-600 bg-primary-100 border-l-2 border-primary-600 py-2 px-3 font-semibold text-sm rounded-sm" href="<%s href %>">
+      <%s title %>
+    </a>
+  | false ->
+    <a class="block hover:text-primary-600 hover:bg-primary-100 hover:border-l-2 hover:border-primary-600 py-2 px-3 font-medium text-sm rounded-sm" href="<%s href %>">
+      <%s title %>
+    </a>
+
+
+let exercises_sidebar_link_group
+?(extra_html="")
+title
+link_html
+=
+   <details class="space-y-2 flex flex-col">
+      <summary class="flex gap-4 justify-between text-sm font-semibold py-2 uppercase focus:text-primary-600 active:text-primary-600 hover:text-primary-300">
+        <%s title %>
+        <%s! Icons.chevron_down "h-6 w-6" %>
+      </summary>
+      <span class="ml-4 border-l"><%s! link_html %></span>
+      <span class="ml-4 border-l"><%s! extra_html %></span>
+    </details>
+
 let exercises_sidebar
 ~exercises
 =
 let open Data in
 let render_exercise_link (exercise : Exercise.t) =
   let href = "#" ^ exercise.number in
-  Learn_layout.sidebar_link ~title:exercise.title ~href ~current:false
+  exercises_sidebar_link ~title:exercise.title ~href ~current:false
 in
 let exercise_sidebar_links_by_tag ?tag title =
-  <%s! Learn_layout.sidebar_link_group title
+  <%s! exercises_sidebar_link_group title
     (Exercise.filter_tag ?tag exercises |> List.fold_left (fun a b -> a ^ render_exercise_link b) "")
   %>
 in
-
+  <p class="font-bold">Exercises Categories</p>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"list" "Lists" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"arithmetic" "Arithmetic" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"logic" "Logic and Codes" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"binary-tree" "Binary Trees" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"multiway-tree" "Multiway Trees" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag ~tag:"graph" "Graphs" %>
+  <hr>
   <%s! exercise_sidebar_links_by_tag "Miscellaneous" %>
```

Comment:
```suggestion
  <div class="pr-3">
  <p class="font-bold mb-4">Exercises Categories</p>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"list" "Lists" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"arithmetic" "Arithmetic" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"logic" "Logic and Codes" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"binary-tree" "Binary Trees" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"multiway-tree" "Multiway Trees" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag ~tag:"graph" "Graphs" %>
    <hr>
    <%s! exercise_sidebar_links_by_tag "Miscellaneous" %>
  </div>
```

There is some right padding, so that the chevrons don't overlap with the scrollbar.